Feature/material design

JeffHeck opened the pull request:
Feel free to review and pull this into develop.

JeffHeck committed:
Init blank branch material_design

JeffHeck committed:
Sample behavior stage.
Ready to review and discuss.

loki980 committed:
Moved background images from mdpi to nodpi to fix an OOM error on a default Nexus 6P AVD.

JeffHeck committed:
Renamed package name and applicationID to: com.lokico.PSWind

JeffHeck committed:
Added 4 simple unit tests that indicate when basic functionality
will be ready for additional tests:
OmniMap, TJsSeattle, TJsNorthSound, NOAA

loki980 commented on app/src/test/java/com/lokico/PSWind/BasicUnitTests.java:
```diff
@@ -0,0 +1,30 @@
+public class BasicUnitTests {
+    @Test
+    public void isNoaaImplemented() {
+        assertEquals(true, NOAA.isActivityImplemented());
```
You can use assertTrue() instead of assertEquals() here.  In this situation, it's just more succinct.  It's actually more important to use assertEquals() instead of doing something like:
`assertEquals(true, NOAA.numOverlaysAvailable() == 3);`

That makes the failure message actually helpful.

JeffHeck committed:
Modified local unit tests:
They now use assertTrue instead of assertEquals

Added Instrumentation tests:
Using Espresso
These tests exercise the buttons on the opening screen.
They also exercise the navigation drawer items.

JeffHeck commented:
Added UI navigation tests with Espresso.
Also have a first cut at the issues from the FogBugz cases.
